backend/src/main/scala/cromwell/backend/standard/StandardFinalizationActor.scala

```diff
@@ -0,0 +1,36 @@
+package cromwell.backend.standard
+
+import cromwell.backend._
+import cromwell.core.CallOutputs
+import wdl4s.TaskCall
+
+trait StandardFinalizationActorParams {
+  val workflowDescriptor: BackendWorkflowDescriptor
```

Comment:
Hah! Yes. I did. Good catch.

Mainly `val`s were painful when the base trait was trying to use a `val myField = <some default>`, later overridden by a sub class / trait. Because these fields are not actually used by the base trait, there was no pain involved… yet?

I'll look at changing them all to `def`s just in case.